licenses/types.go

```diff
@@ -0,0 +1,328 @@
+	"BSD-3-Clause-Clear":            Notice,
+	"BSD-3-Clause-LBNL":             Notice,
+	"BSD-3-Clause-OpenMPI":          Notice,
+	"BSD-4-Clause":                  Notice,
+	"BSD-4-Clause-UC":               Notice,
+	"BSD-Protection":                Notice,
+	"BSL-1.0":                       Notice,
+	"CC-BY-1.0":                     Notice,
+	"CC-BY-2.0":                     Notice,
+	"CC-BY-2.5":                     Notice,
+	"CC-BY-3.0":                     Notice,
+	"CC-BY-4.0":                     Notice,
+	"FTL":                           Notice,
+	"ISC":                           Notice,
+	"ImageMagick":                   Notice,
+	"LPL-1.0":                       Notice,
+	"LPL-1.02":                      Notice,
+	"Libpng":                        Notice,
+	"Lil-1.0":                       Notice,
+	"Linux-OpenIB":                  Notice,
+	"MIT":                           Notice,
+	"MS-PL":                         Notice,
+	"NCSA":                          Notice,
+	"OpenSSL":                       Notice,
+	"PHP-3.0":                       Notice,
+	"PHP-3.01":                      Notice,
+	"PIL":                           Notice,
+	"PostgreSQL":                    Notice,
+	"Python-2.0":                    Notice,
+	"Python-2.0-complete":           Notice,
+	"SGI-B-1.0":                     Notice,
+	"SGI-B-1.1":                     Notice,
+	"SGI-B-2.0":                     Notice,
+	"UPL-1.0":                       Notice,
+	"Unicode-DFS-2015":              Notice,
+	"Unicode-DFS-2016":              Notice,
+	"Unicode-TOU":                   Notice,
+	"W3C":                           Notice,
+	"W3C-19980720":                  Notice,
+	"W3C-20150513":                  Notice,
+	"X11":                           Notice,
+	"Xnet":                          Notice,
+	"ZPL-1.1":                       Notice,
+	"ZPL-2.0":                       Notice,
+	"ZPL-2.1":                       Notice,
+	"Zend-2.0":                      Notice,
+	"Zlib":                          Notice,
+	"zlib-acknowledgement":          Notice,
+
+	"BSD-0-Clause": Unencumbered,
+	"CC0-1.0":      Unencumbered,
+	"Unlicense":    Unencumbered,
+
+	"AGPL-1.0":          Forbidden,
+	"AGPL-3.0":          Forbidden,
+	"CC-BY-NC-1.0":      Forbidden,
+	"CC-BY-NC-2.0":      Forbidden,
+	"CC-BY-NC-2.5":      Forbidden,
+	"CC-BY-NC-3.0":      Forbidden,
+	"CC-BY-NC-4.0":      Forbidden,
+	"CC-BY-NC-ND-1.0":   Forbidden,
+	"CC-BY-NC-ND-2.0":   Forbidden,
+	"CC-BY-NC-ND-2.5":   Forbidden,
+	"CC-BY-NC-ND-3.0":   Forbidden,
+	"CC-BY-NC-ND-4.0":   Forbidden,
+	"CC-BY-NC-SA-1.0":   Forbidden,
+	"CC-BY-NC-SA-2.0":   Forbidden,
+	"CC-BY-NC-SA-2.5":   Forbidden,
+	"CC-BY-NC-SA-3.0":   Forbidden,
+	"CC-BY-NC-SA-4.0":   Forbidden,
+	"Commons-Clause":    Forbidden,
+	"Facebook-2-Clause": Forbidden,
+	"Facebook-3-Clause": Forbidden,
+	"Facebook-Examples": Forbidden,
+	"WTFPL":             Forbidden,
+
+	"AML":                                Unknown,
```

Comment:
Why are so many licenses unknown type?

Reply:
These are all the `licenseclassifier/v2` licenses, I haven't found any mappings for them (eg. in v1).
If you know how to map these, we can still move them from `Unkown` to another category.
It's just that I don't know where these belong.

Reply:
Ohh, that's a lot of work!
I think this is non blocking for merge, because go-licenses could not identify these licenses before.